src/src/tx_main.cpp

```diff
@@ -493,6 +548,8 @@ void ICACHE_RAM_ATTR timerCallbackNormal()
     TelemetryRcvPhase = ttrpTransmitting;
   }
 
+  BeginClearChannelAssessment();
```

Comment:
Shouldnt this be called TXdoneISR to maximise the time before sending the next packet?

Reply:
Yes, that would be much better. Just have to wrap my head around if there are any corner cases where disabling RX interrupt and then rely on it being enabled again in another interrupt can go wrong?

Reply:
Implemented, got rid of busy waiting.